### ISSUE TITLE
Description of run types

### ISSUE THREAD

**Issue by AloisSeckar** [AloisSeckar/WBSC-Scoring#224]
Currently we only show run types abbreviations (ER, UE, TU) in UI:

![image](https://github.com/AloisSeckar/WBSC-Scoring/assets/7399922/1a7f8c46-387e-4ae3-b164-0b3f34a1fb54)

It might be worth adding some kind of explanation or hints especially for translations to other languages as suggested in #221

**Comment by lucafano04** [comment]
Suggestion - leave the shorts name there and place ER/UE/TU inside the `/help` page next to the:
> Then you pick from "Earned" / "Unearned" / "Team unearned" options

**Comment by AloisSeckar** [comment]
Good idea. It is online in testing version - https://wbsc-scoring-dev.netlify.app/
